natives.json

```diff
@@ -2261,11 +2249,11 @@
       "comment": "",
       "params": [
         {
-          "type": "Any",
-          "name": "p0"
+          "type": "const char*",
+          "name": "ActorSetName"
```

Comment:
To maintain consistency with our coding standards, we require that all parameter names use camelCase. Parameters should not start with capital letters. Update the code to reflect this convention.